10x/count/count.wdl

```diff
@@ -404,7 +415,7 @@ task align_reads_split {
     read_groups_json='["${sep='","' read_groups}"]'
     echo "$(jq --arg var "$read_groups_json" '. + {"read_groups": ( $var | fromjson )}' _args)" > _args
     echo "$(jq --arg var "[]" '. + {"read2s": ( $var | fromjson )}' _args)" > _args
-    echo "$(jq --arg var "${reference_path}" '. + {"reference_path": $var}' _args)" > _args
+    echo "$(jq --arg var "genome_reference" '. + {"reference_path": $var}' _args)" > _args  # todo no required changes
```

Comment:
what does the todo mean? it sounds like it's a todo to not do anything :)

Reply:
You're right; it was a note to myself that I didn't remove. Thanks for the catch.